docs/8-kubernetes-container-orchestration/8.6-webhooks.md

```diff
@@ -117,12 +121,16 @@ Controller using python.
     cat validating-webhook.yaml | sed "s/      caBundle: .*/      caBundle: ${CA}/" | kubectl apply -f -
     ```
 
+?> It might be in your best interest to create scripts or "one liners" to deploy and delete resources as you may do this a lot
+
 8. Test your validating webhook
 Tests 2, 4, and 5 should pass.
```

Comment:
change this with corresponding logic that is provided for validation for the test pods

Reply:
I added a note that poses a question to the developer